Description: Customize keybinding for scrolling page 
Author: Piotr 
Origin: <upstream|backport|vendor|other>, <URL, required except if Author is present>
Bug: <URL to the upstream bug report if any, implies patch has been forwarded, optional>
Bug-<Vendor>: <URL to the vendor bug report if any, optional>
Forwarded: <URL|no|not-needed, useless if you have a Bug field, optional>
Applied-Upstream: <version|URL|commit, identifies patches merged upstream, optional>
Reviewed-by: <name and email of a reviewer, optional>
Last-Update: 2017-04-21 <YYYY-MM-DD, last update of the meta-information, optional>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/ui/pageview.cpp
+++ b/ui/pageview.cpp
@@ -1724,8 +1724,8 @@ void PageView::keyPressEvent( QKeyEvent
     // move/scroll page by using keys
     switch ( e->key() )
     {
-        case Qt::Key_J:
-        case Qt::Key_K:
+        case Qt::Key_T:
+        case Qt::Key_C:
         case Qt::Key_Down:
         case Qt::Key_PageDown:
         case Qt::Key_Up:
@@ -1733,14 +1733,14 @@ void PageView::keyPressEvent( QKeyEvent
         case Qt::Key_Backspace:
             if ( e->key() == Qt::Key_Down
                  || e->key() == Qt::Key_PageDown
-                 || e->key() == Qt::Key_J )
+                 || e->key() == Qt::Key_T )
             {
-                bool singleStep = e->key() == Qt::Key_Down || e->key() == Qt::Key_J;
+                bool singleStep = e->key() == Qt::Key_Down || e->key() == Qt::Key_T;
                 slotScrollDown( singleStep );
             }
             else
             {
-                bool singleStep = e->key() == Qt::Key_Up || e->key() == Qt::Key_K;
+                bool singleStep = e->key() == Qt::Key_Up || e->key() == Qt::Key_C;
                 slotScrollUp( singleStep );
             }
             break;
@@ -1756,6 +1756,7 @@ void PageView::keyPressEvent( QKeyEvent
                 horizontalScrollBar()->triggerAction( QScrollBar::SliderSingleStepSub );
             break;
         case Qt::Key_Right:
+        case Qt::Key_N:
         case Qt::Key_L:
             if ( horizontalScrollBar()->maximum() == 0 )
             {
